zask/logging.py

```diff
@@ -35,9 +35,13 @@ def debug_handler():
 
 
 def production_handler(config):
-    handler = RotatingFileHandler(config['ERROR_LOG'],
-                                  maxBytes=1024 * 50,
-                                  backupCount=5)
+    if config.get('PRODUCTION_LOGGING_CLASS') == 'WatchedFileHandler':
```

Comment:
? `PRODUCTION` is a little confusing, maybe just call it `LOGGING_HANDLER`?

Reply:
There is another setting: `PRODUCTION_LOGGING_LEVEL`, I try to maintain consistency.